sdk/identity/azure-identity/test/ut/default_azure_credential_test.cpp

```diff
@@ -195,46 +195,49 @@ TEST(DefaultAzureCredential, LogMessages)
             "ClientSecretCredential with corresponding tenantId, clientId, clientSecret, and "
             "authorityHost gets created.");
 
-        EXPECT_EQ(log[3].first, Logger::Level::Verbose);
-        EXPECT_EQ(
-            log[3].second,
-            "Identity: ManagedIdentityCredential: Environment is not set up for the credential "
-            "to be created with App Service 2019 source.");
+        EXPECT_EQ(log[3].first, Logger::Level::Informational);
+        EXPECT_EQ(log[3].second, "Identity: WorkloadIdentityCredential was created successfully.");
 
         EXPECT_EQ(log[4].first, Logger::Level::Verbose);
         EXPECT_EQ(
             log[4].second,
             "Identity: ManagedIdentityCredential: Environment is not set up for the credential "
-            "to be created with App Service 2017 source.");
+            "to be created with App Service 2019 source.");
```

Comment:
The diff might seem confusing, but all that's changing is a log item is being inserted at log[3], and hence all other log items get shifted down (so log[4] becomes log[5] etc.).